server/controllers/request.js

```diff
@@ -99,3 +101,65 @@ exports.updateRequest = asyncHandler(async (req, res, next) => {
     }
   });
 });
+
+exports.payPetSitter = asyncHandler(async (req, res, next) => {
+  let { 
+    userId, 
+    customerId, 
+    sitterId,
+    email, 
+    name, 
+    serviceId, 
+    hourlyRate, 
+    hours,
+    successUrl,
+    cancelUrl
+  } = req.body;
+
+  const orderId = req.params.id;
+
+  try {  
+    if (customerId === "") {
+      const customer = await stripe.customers.create({email, name});
+      await Profile.updateOne({_id: userId}, {customerId: customer.id});
+      customerId = customer.id;
+    }
```

Comment:
Same as below for here as well

Reply:
I could create a new Stripe customer when a user registers. That's right?